hooks/templatedataformatter.js

```diff
@@ -27,6 +27,7 @@ module.exports = function (pageMetadata, siteLevelAttributes, pageNameToConfig)
     relativePath,
     env: {
       JAMBO_INJECTED_DATA: env.JAMBO_INJECTED_DATA,
+      IS_DEVELOPMENT_PREVIEW: !!env.IS_DEVELOPMENT_PREVIEW,
```

Comment:
`!!` could be incorrect if somehow IS_DEVELOPMENT_PREVIEW set to "false" right? Maybe IS_DEVELOPMENT_PREVIEW === 'true' is better

Reply:
ah yeah that's true

Reply:
it's weird we used a string, I think looking back I would have asked it to be 0 or 1